src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -27,7 +26,7 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         }
 
         model.redoRestaurantBook();
-        model.updateFilteredItemList(PREDICATE_SHOW_ALL_ITEMS, Member.class);
+        model.updateFilteredMemberList(PREDICATE_SHOW_ALL_MEMBERS);
```

Comment:
(Out of topic) - Might need to update all the filtered lists once we implement our own filter operations.

Reply:
Yes